Makefile

```diff
@@ -29,7 +29,9 @@ component-test:
 
 #Pushes the container to the docker registry/repository.
 publish: package
-	@docker push $(PUBLISH_TAG)
+	docker tag meetup/gauth-proxy:latest $(PUBLISH_TAG)
```

Comment:
I dont recommend to use `latest` in the client side. But it should be good just pushing this alias.